### PR TITLE
Added api version check

### PR DESCRIPTION
**Pull request by danderson00**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@danderson00__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dalander). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by danderson00** [comment]
Resolves https://github.com/Azure/azure-mobile-apps-node/issues/90

File: src/express/index.js [hunk 1]
```diff
@@ -38,27 +39,20 @@ var express = require('express'),
...
+        .use(authenticate(configuration))
+        .use(crossOrigin(configuration))
         .use(customMiddlewareRouter)
-        .use(notificationMiddleware)
+        .use(configuration.notificationRootPath || '/push/installations', notifications(configuration))
         .use(configuration.apiRootPath || '/api', apiMiddleware)
-        .use(configuration.tableRootPath || '/tables', tableMiddleware, renderResults)
-        .use(handleErrorMiddleware);
+        .use(configuration.tableRootPath || '/tables', apiVersionCheck(configuration), tableMiddleware, renderResults)
+        .use(handleError(configuration));
```
Comment (mamaso) [inline src/express/index.js]: good call, cleaner

File: src/express/middleware/apiVersionCheck.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var acceptedVersionRegex = /^2[.]0[.]\d+$/,
```
Comment (mamaso) [inline src/express/middleware/apiVersionCheck.js]: missing header

Comment (danderson00) [inline src/express/middleware/apiVersionCheck.js]: Good spot. Added.

**Pull request merged**